server/services/offendersService.js

```diff
@@ -775,6 +838,15 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
           const status = sortByStatus(b.dbStatus, a.dbStatus)
           return status === 0 ? sortByDateTime(b.nextReviewDateDisplay, a.nextReviewDateDisplay) : status
         })
+        .map(o => {
+          return {
+            ...o,
+            securityReferred:
+              securityReferredOffenders
+                .filter(s => s.offenderNo === o.offenderNo)
+                .filter(s => s.status === 'NEW' || s.status === 'REFERRED').length > 0,
```

Comment:
Restricted them for the label on the recat page, as I thought it didn't make much sense to display tag for a cancelled referral?

Reply:
maybe .some() is preferable here